report/src/main/java/com/hartwig/actin/report/interpretation/EvaluatedTrialComparator.java

```diff
@@ -11,26 +11,6 @@ public class EvaluatedTrialComparator implements Comparator<EvaluatedTrial> {
 
     private static final String COMBINATION_COHORT_IDENTIFIER = "+";
 
-    @Override
-    public int compare(@NotNull EvaluatedTrial trial1, @NotNull EvaluatedTrial trial2) {
-        int hasMolecularCompare = Boolean.compare(trial1.molecularEvents().isEmpty(), trial2.molecularEvents().isEmpty());
-        if (hasMolecularCompare != 0) {
-            return hasMolecularCompare;
-        }
-
-        int trialCompare = trial1.trialId().compareTo(trial2.trialId());
-        if (trialCompare != 0) {
-            return trialCompare;
-        }
-
-        int cohortCompare = compareCohorts(!trial1.molecularEvents().isEmpty(), trial1.cohort(), trial2.cohort());
-        if (cohortCompare != 0) {
-            return cohortCompare;
-        }
-
-        return compareSets(trial1.molecularEvents(), trial2.molecularEvents());
-    }
-
```

Comment:
Currently public functions are always at the top of the class, where private static follow towards the bottom. 

Is there a specific reason why you swapped the ordering here?

Reply:
This is from running `Reformat` in IntelliJ after importing the code style settings. It seems to group static functions at the top before considering public/private. Do you get a different result in your IDE? I'll take a look at my settings.

Reply:
The reason seems to be that our scheme is set up for `Reformat` only and not for `Rearrange`